src/shared/util.js

```diff
@@ -310,6 +310,28 @@ const PasswordResponses = {
   INCORRECT_PASSWORD: 2,
 };
 
+const EVENT_MAP = {
```

Comment:
Can we please use a more descriptive, and less general, name here?

How about something like e.g. `AnnotationActionEventType` or something, which is more consistent with other Annotation types defined earlier in this file?